src/commands/qrcode.cpp

```diff
@@ -23,25 +23,172 @@
 #include <iostream>
 #include <bitcoin/explorer/define.hpp>
 
+#if defined WITH_QRENCODE && defined WITH_LIBPNG
```

Comment:
It would seem that there are two useful forms of the qrcode, and these correspond to independent dependencies. If these are useful independently, which I think they are, it would be better to expose this functionality as an option and decouple the two dependencies. IOW one may ask for a qrcode stream OR a png encoding of a qrcode stream. One does not require the png dependency if one is after the former only.

I would accomplish this by adding a Boolean (unvalued) option named `--image`, which if present changes the serialization to `png`. We could eventually change this to `--format=[value]` where value is an enumerated type. This takes a little more work in terms of defining the enumeration primitive, and given it requires more dependencies I would stick with the Boolean for now.